omnihub/src/main/java/org/eclipse/deeplearning4j/omnihub/api/ModelType.java

```diff
@@ -17,11 +17,21 @@
  *  * SPDX-License-Identifier: Apache-2.0
  *  *****************************************************************************
  */
+package org.eclipse.deeplearning4j.omnihub.api;
 
-package org.nd4j.codegen.api
+public enum ModelType {
+    COMP_GRAPH,SEQUENTIAL;
 
-enum class Namespace {
-    BASE, CNN, IMAGE, LOSS, MATH, NN, RANDOM, RNN;
+    public static ModelType fromString(String namespace) {
+        switch(namespace.toLowerCase()) {
+            case "comp_graph":
+                return COMP_GRAPH;
+            case "sequential":
+                return SEQUENTIAL;
+            default:
+                return null;
+        }
+    }
 
-    fun displayName() = name[0].toString() + name.substring(1).toLowerCase()
-}
\ No newline at end of file
+
+}
```

Comment:
This change looks out of place. Have you accidentally moved that file and then renamed it or something like that?

Reply:
I'm saying this because it is shown as a move and rename:
` .../kotlin/org/nd4j/codegen/api/Namespace.kt → ...deeplearning4j/omnihub/api/ModelType.java ` 

And the Namespace.kt file is no longer in place

Reply:
@treo actually namespace isn't used anywhere so I just deleted it. The code compiles just fine without it. The namespace generation also runs just fine without this present. The move was originally because I was trying to adapt some of the codegen infra to the omnihub SDK and just noticed what was unused.